nuxeo-csv-core/src/test/java/org/nuxeo/ecm/csv/core/TestCSVImporterCreateMode.java

```diff
@@ -473,13 +474,22 @@ public void shouldSetCreatorToTheUserImporting() throws InterruptedException {
 
         try (CloseableCoreSession leelaSession = coreFeature.openCoreSession("leela")) {
 
+            // NXP-22172 : appears when leela does not have access to "/", yet ?
+            if (!session.exists(new PathRef("/"))) {
```

Comment:
I don't understand why session will not see the root, shouldn't be leelaSession.exists instead ?

Reply:
Yes